jib-core/src/test/java/com/google/cloud/tools/jib/api/JibContainerBuilderTest.java

```diff
@@ -139,7 +139,8 @@ public void testToBuildConfiguration()
             .setBaseImageLayersCache(Paths.get("base/image/layers"))
             .setApplicationLayersCache(Paths.get("application/layers"))
             .setExecutorService(mockExecutorService)
-            .addEventHandler(mockJibEventConsumer);
+            .addEventHandler(mockJibEventConsumer)
+            .setAlwaysCacheBaseImage(true);
```

Comment:
ditto

Reply:
I instead changed it to `.setAlwaysCacheBaseImage(false)` since asserting 
`Assert.assertFalse(buildConfiguration.getAlwaysCacheBaseImage())`.